invenio_app_rdm/theme/templates/invenio_app_rdm/search.html

```diff
@@ -6,28 +6,22 @@
     under the terms of the MIT License; see LICENSE file for more details.
 #}
 
-{% extends "invenio_search_ui/search.html" %}
-
-{%- block page_header %}
-{% include config.THEME_HEADER_TEMPLATE %}
-{%- endblock page_header %}
-
-{%- block search_loading %}{% endblock %}
-
-{%- block search_sort_select %}
-Sort by:
-<invenio-search-select-box
-   sort-key="sort"
-   available-options='{{ sort_options|format_sortoptions|safe }}'
-   template="{{ url_for('static', filename=config.SEARCH_UI_JSTEMPLATE_SELECT_BOX) }}"
-  >
-</invenio-search-select-box>
-{%- endblock search_sort_select %}
-
-{%- block javascript %}
+<!DOCTYPE html>
+<html>
+  <head>
+    {%- block css %}
+    {{ webpack['vendor.css']}}
```

Comment:
I guess these can be extracted at some point into an RDM-specific base page.html template

Reply:
@slint no for now because in the `vendor.css` semantic-ui is included while in the module specific css, e.g `invenio-app-rdm-theme.css` bootstrap is included. As long as we don't have a clear mechanism to separate these 2 currently, I need to include `vendor.css` in the dedicated search page. But in the long run I agree with you about the common rdm page.html.